App/templates/App/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title> USER CONTRIBUTION SUMMARY </title>
+</head>
+<body>
+		<center>
+	     <form method="post"> 
+	     	{% csrf_token %}
+	     	Enter Username: <input type="text" name="username">
+	     	<br>
+	     	<br>
+	     	<button type='submit'> Submit </button>
+	     </form>
+	    </center> 
+	    
+	    <h2>
+	    {% if userid %}
+	    	Username = {{username}} <br>
+	    	Userid = {{userid}} <br>
+	    	<a href="https://phabricator.wikimedia.org/p/{{username}}/"> Profile link </a>
+		<br>
+		<br>
+		<h1>English Edits- </h1><br>
+	    {% endif %}
+		</h2>
+	    {% for x in contribution %}  <!-- Here I traverse all the contributions and display the following div-->
+	    <div>
+	    	Title = {{x.title}} <br>
+	    	PageID = {{x.pageid}} <br>
+	    	RevisionID = {{x.revid}} <br>
+	    	ParentID = {{x.parentid}} <br>
+	    	ns = {{x.ns}} <br>
+	    	Timestamp = {{x.timestamp}} <br>
+	    	Comment = {{x.comment}} <br>
+	    	Size = {{x.size}} <br>
+
+	    	<a href="https://en.wikipedia.org/w/index.php?title=User:{{x.user}}&diff=prev&oldid={{x.revid}}">
```

Comment:
Technically, that's not valid HTML (which is why Github shows it in red); special characters like `&` or `"` need to be encoded in attributes. So it should be `{{x.user}}&amp;diff`. In practice, it works anyway and no one really cares about, so feel free to ignore :) just mentioning for sake of completeness.

Reply:
Rectified .